x-pack/plugins/files/tsconfig.json

```diff
@@ -6,9 +6,18 @@
     "declaration": true,
     "declarationMap": true
   },
-  "include": ["common/**/*", "public/**/*", "server/**/*"],
+  "include": [
+    "common/**/*",
+    "public/**/*",
+    "server/**/*",
+    ".storybook/**/*"
+  ],
   "references": [
-    { "path": "../../../src/core/tsconfig.json" },
-    { "path": "../security/tsconfig.json" },
+    {
+      "path": "../../../src/core/tsconfig.json"
+    },
+    {
+      "path": "../security/tsconfig.json"
+    },
   ]
-}
+}
```

Comment:
nit: \\n

```suggestion
}

```

Reply:
Ah, I'd turned of all my extensions and hadn't turned on the correct JSON formatter yet. Well spotted 😄